microcosm_elasticsearch/tests/test_mapping.py

```diff
@@ -1,242 +0,0 @@
-"""
-Test that the way we define a SearchIndex, Store and Model is consistent when overriding defaults
```

Comment:
@jessemyers I think that's only partially true - the first tests check that we can override the default `/doc` for the mapping name. In practice we're not using that possibility so we can just remove it (and it's going away in ES7 anyway, where all mappings will be under `_doc`.
The last test (`"Results are cast to the correct class according to their doctypes"`) is actually fairly important, since it ensures that we can have polymorphic types in a single mapping/index, using the `doctype` property to cast each result to the right model.